mixpanel.js

```diff
@@ -2339,15 +2339,15 @@ Globals should be all caps
     };
 
     /**
-     * Track an event.  This is the most important Mixpanel function and
-     * the one you will be using the most.
+     * Track an event.  This is the most important and
```

Comment:
@tdumitrescu these double spaces have actually always been there. want me to remove them throughout?